orangecontrib/text/widgets/tests/test_owcorpus.py

```diff
@@ -286,6 +287,40 @@ def test_keep_selected_variables(self):
         self.wait_until_finished()
         self.assertListEqual(list(prew_selected), self.widget.used_attrs)
 
+    def test_no_text_feature(self):
+        """
+        Test with data which have empty text_features. Widget should not show
+        the error but, should have all features unused.
+        """
+        # widget already loads book-excerpts from file and store context
+        # settings this call restore context settings to default otherwise
+        # Text variable is moved to used_attributes by the context
+        self.widget.settingsHandler.reset_to_original(self.widget)
```

Comment:
@ajdapretnar do you maybe know is there a simpler way to reset settings?

Reply:
Whoa, no idea. It looks ok to me. 🤷